OpenChange/MAPIStoreMailMessage.m

```diff
@@ -572,28 +572,21 @@ - (int) getPidTagMessageClass: (void **) data
   return MAPISTORE_SUCCESS;
 }
 
-/* Note: this applies to regular mails... */
-// - (int) getPidTagReplyRequested: (void **) data // TODO
-//                    inMemCtx: (TALLOC_CTX *) memCtx
-// {
-//   if (!headerSetup)
-//     [self _fetchHeaderData];
-
-//   return (mailIsEvent
-//           ? [self getYes: data inMemCtx: memCtx]
-//           : [self getNo: data inMemCtx: memCtx]);
-// }
-
-/* ... while this applies to invitations. */
-- (int) getPidTagResponseRequested: (void **) data // TODO
-                          inMemCtx: (TALLOC_CTX *) memCtx
+- (int) getPidTagReplyRequested: (void **) data
+		       inMemCtx: (TALLOC_CTX *) memCtx
 {
   if (!headerSetup)
     [self _fetchHeaderData];
 
   return (mailIsEvent
-          ? [self getNo: data inMemCtx: memCtx]
-          : MAPISTORE_ERR_NOT_FOUND);
+          ? [self getYes: data inMemCtx: memCtx]
+          : [self getNo: data inMemCtx: memCtx]);
```

Comment:
Only return Yes when the mail is a meeting request/response. A mail is an event if it has icalendar MIME type, so it includes every type of calendar.

With regards to returning No or NOT FOUND, it is just unneccesary to change but I'm ok if you want to use it for some reason I don't see right now.